src/systems/filecoin_vm/sysactors/init_actor.go

```diff
@@ -72,6 +72,8 @@ func (a *InitActorCode_I) Exec(rt Runtime, codeID actor.CodeID, constructorParam
 		rt.AbortArgMsg("cannot exec an actor of this type")
 	}
 
+	rt.CreateActor_DeductGas()
```

Comment:
It already calls `rt.CreateActor`: move the charge there and simplify the Runtime interface.